FreeTubeApp/FreeTube
# Add video reload button

## Pull Request Type


## Related issue



Closes #1578 (Most code changes comes from it)

## Description

Add reload on top nav only visible on watch video page
Press = reload video player (render a new one)

## Screenshots 


https://github.com/FreeTubeApp/FreeTube/assets/1018543/12766393-ecc0-4e7c-ad32-e3763b1c545b


## Testing 



- Open random video/live stream/upcoming stuff (with/without playlist)
- Seek to random time
- Press reload
- Video should play with last timestamp (if it can be played)

## Desktop

- **OS:**
- **OS Version:**
- **FreeTube version:**

## Additional context
